app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET vNext Sample app",
+    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
+    "env": {
+	"BUILDPACK_URL": "https://github.com/friism/heroku-buildpack-mono#aspnet-vnext"
```

Comment:
Hi there! Do you know if there's an official Heroku build thingy (a technical term!) for ASP.NET 5? We generally prefer not to link to personal accounts due to support and reliability issues.